.github/workflows/ci-integration.yml

```diff
@@ -553,6 +558,7 @@ jobs:
   # Repo owner has commented /ok-to-test on a (fork-based) pull request
   integration-fork-windows:
     runs-on: windows-latest
+    name: Windows-integration-forked
```

Comment:
I will take a look and test it

Reply:
I don't really find a -ignore flag online. And by ignore what do you mean? I think currently we have the if statement to check whether we should skip the job or not

Reply:
Just so we know which flows we should ignore (the empty ok-to-test ones), I also couldn't find how we can view the logs of the workflow (besides going directly to git actions), any thoughts?

Reply:
on ok-to-test.yaml line 12. We can uncomment that line. But still any comment on a pull-request would trigger this workflow. It will run quickly to check whether it is a /ok-to-test command. Are u suggesting that we should only trigger this wf when someone commented /ok-to-test? TBH i think that is impossible. The workflow will run anyway. All we can do is add if statement to decide whether this workflow is skipped or completed. 

For example, if u delete line 12. Now a comment on an issue will still trigger this workflow, however it will be displayed as skipped. 
If u comment something other than /ok-to-test, it will trigger this workflow as well, but it will be displayed as completed with a green check, but actually does nothing.

Reply:
I am still doing research tho

Reply:
Working on the log issue. I think there might be a way to send the log along with the check status to the pull request. I am testing right now.